input/fsh/who-co-Observations.fsh

```diff
@@ -0,0 +1,294 @@
+CodeSystem: WhoCoPositiveNegativeUnknown
+Title: "Positive/Negative/Unknown"
+Description: "Positive/Negative/Unknown"
+* #positive
+  "Positive"
+* #negative
+  "Negative"
+* #unknown
+  "Unknown"
+
+ValueSet: WhoCoValueSetPositiveNegativeUnknown
+Id: who-co-positive-negative-unknown
+Title: "Positive/Negative/Unknown"
+Description: "Positive/Negative/Unknown"
+* codes from system WhoCoPositiveNegativeUnknown
+
+CodeSystem: WhoCoHealthOutcome
+Title: "Health Outcome"
+Description: "Health Outcome"
+* #recovered_healthy
+  "Recovered/Healthy"
+* #not_recovered
+  "Not Recovered"
+* #death
+  "Death"
+* #unknown
+  "Unknown"
+* #other
+  "Other"
+
+ValueSet: WhoCoValueSetHealthOutcome
+Id: who-co-health-outcome
+Title: "Health Outcome"
+Description: "WH Co Valueset Health Outcome"
+* codes from system WhoCoHealthOutcome
```

Comment:
Should these maybe go in the existing terminology file? Just to keep all related structure definitions together and to allow for reuse.
I'm not sure...
@rcrichton

Reply:
@MattyJ007 do you think they should be in the existing file or a new file specific to `Case Outcome`?

Reply:
In the existing file should be fine - these options seem generic